syntax = "proto2";

package opentxs.proto;
option optimize_for = LITE_RUNTIME;

import public "StorageItemHash.proto";

message StorageNymList {
    optional uint32 version = 1;
    repeated StorageItemHash nym = 2;
}
